react-hot-button-component/src/hot-button.jsx

```diff
@@ -1,30 +1,35 @@
 import React, { useState } from 'react';
 
 export function HotButton({ text }) {
-  const [index, setIndex] = useState(0);
+  const [clicks, setClicks] = useState(0);
 
   function handleClick() {
-    setIndex(index + 1);
+    setClicks(clicks + 1);
   }
-  if (index < 3) {
-    return <button onClick={handleClick} className="black">{text}</button>;
-  }
-  if (index >= 3 && index < 6) {
-    return <button onClick={handleClick} className="purple">{text}</button>;
-  }
-  if (index >= 6 && index < 9) {
-    return <button onClick={handleClick} className="light-purple">{text}</button>;
-  }
-  if (index >= 9 && index < 12) {
-    return <button onClick={handleClick} className="orange">{text}</button>;
-  }
-  if (index >= 12 && index < 15) {
-    return <button onClick={handleClick} className="peach">{text}</button>;
-  }
-  if (index >= 15 && index < 18) {
-    return <button onClick={handleClick} className="yellow">{text}</button>;
-  }
-  if (index >= 18) {
-    return <button onClick={handleClick} className="white">{text}</button>;
+
+  function setClassName() {
+    if (clicks < 3) {
+      return 'black';
+    }
+    if (clicks >= 3 && clicks < 6) {
+      return 'purple';
+    }
+    if (clicks >= 6 && clicks < 9) {
+      return 'light-purple';
+    }
+    if (clicks >= 9 && clicks < 12) {
+      return 'orange';
+    }
+    if (clicks >= 12 && clicks < 15) {
+      return 'peach';
+    }
+    if (clicks >= 15 && clicks < 18) {
+      return 'yellow';
+    }
+    if (clicks >= 18) {
```

Comment:
This check is unnecessary. You could replace lines 29-30 with just `return 'white';`